### ISSUE TITLE
1.7.9 segfaults under Wayland (Sway), 1.7.7 works fine

### ISSUE THREAD

**Issue by alex3kov** [telegramdesktop/tdesktop#6170]
### Steps to reproduce
1. Start telegram-desktop 1.7.9
2. Get segfault.
3. Install telegram-desktop 1.7.7
4. Observe working telegram-desktop

### Expected behaviour
Working telegram-desktop

### Actual behaviour
1.7.9:
```
> telegram-desktop 
QApplication: invalid style override passed, ignoring it.
    Available styles: qt5ct-style, Windows, Fusion
Using Wayland-EGL
Using the 'xdg-shell' shell integration
Segmentation fault (core dumped)
```
1.7.7:
```
> telegram-desktop 
QApplication: invalid style override passed, ignoring it.
    Available styles: qt5ct-style, Windows, Fusion
Using Wayland-EGL
Using the 'xdg-shell' shell integration
```
### Configuration
**Operating system:** Parabola (Arch Linux without non-free packages). 

**Version of Telegram Desktop:** 1.7.9

**Used theme**: none

**WM**: Sway with `xwayland disable` in config. 1.7.7 and earlier versions were working fine for a long time for me with this setup.

**Comment by aDogCalledSpot** [comment]
I'm having the same issue under X with i3. Are you by any chance using qt5ct? I'm using it and the Arch Wiki states that you should have QT_QPA_PLATFORMTHEME='qt5ct' set. This works fine for all KDE apps but judging by the output here it wants 'qt5ct-style' instead.

**Comment by yan12125** [comment]
telegram-desktop 1.7.9-2 was broken, and [the issue](https://bugs.archlinux.org/task/62977) is fixed since 1.7.9-3. Could you check if your system is up-to-date?

**Comment by aDogCalledSpot** [comment]
1.7.9-3 fixes it for me.
